src/main/java/pl/poznan/put/cs/scenariochecker/transformations/ScenarioHelper.java

```diff
@@ -14,4 +16,15 @@ private ScenarioHelper() {
     public static boolean isSpecialStep(Step step) {
         return step.getContent().startsWith("IF") || step.getContent().startsWith("FOR EACH");
     }
-}
+
+    /**
+     * @param step, list of actors
+     * @return true when step is step without Actor's
+     */
+    public static boolean isNoActorStep(Step step, List<String> actors) {
+        for (String actor : actors) {
+            if (step.getContent().contains(actor)) return false;
```

Comment:
Shouldn't we actually check that a step begins with actor's name here? Instead of checking whether it contains actor's name. @plutasnyy